src/core/models/media_factory.py

```diff
@@ -104,13 +109,13 @@ def make_from_url(video_url):
                 print("Error downloading video:", e)
                 raise Exception("Error Downloading Video")
         else:
-            bucket_name = VideoFactory.aws_bucket
+            bucket_name = AWSS3Utils.aws_bucket
             file_key = video_url
             file_name = file_key.split("/")[-1]
```

Comment:
s3 file names can be of two types = "filename" or "folder/filename", lets handle both situations. Also write a unit test to ensure we can handle both.

Reply:
will do this!

Reply:
I should name the file as `folder/filename` right?
currently I am naming it as filename

Reply:
yes `folder/filename` is correct